COMPATIBILITY.md

```diff
@@ -79,6 +79,8 @@ Newly filed issues should bear the label `pipeline` for ease of tracking.
 - [ ] `HockeyappRecorder` (`hockeyapp`): [JENKINS-33310](https://issues.jenkins-ci.org/browse/JENKINS-33310)
 - [X] `WsCleanup` (`ws-cleanup`): supported as of 0.30
 - [ ] `XCodeBuilder` (`xcode-plugin`): [JENKINS-33355](https://issues.jenkins-ci.org/browse/JENKINS-33355)
+- [ ] `Release Plugin` (`release`): [JENKINS-40765](https://issues.jenkins-ci.org/browse/JENKINS-40765)
```

Comment:
This belongs in the custom step section, according to the proposal in https://github.com/jenkinsci/release-plugin/pull/22.